shop/src/main/java/ro/msg/learning/shop/service/OrderService.java

```diff
@@ -0,0 +1,36 @@
+package ro.msg.learning.shop.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import ro.msg.learning.shop.domain.*;
+import ro.msg.learning.shop.dto.OrderCreateDto;
+import ro.msg.learning.shop.mapper.ProductMapper;
+import ro.msg.learning.shop.repository.CustomerRepository;
+import ro.msg.learning.shop.repository.OrderRepository;
+import ro.msg.learning.shop.strategy.LocationSelectionStrategy;
+
+@Service
+public class OrderService {
+    public static final String CUSTOMER_NOT_FOUND = "Customer not found!";
+    public static final String NO_LOCATION_WITH_SUFFICIENT_STOCK_FOUND = "No location with sufficient stock found";
+    public static final String PRODUCT_NOT_FOUND_WITH_ID = "Product not found with ID: ";
+    @Autowired
+    private OrderRepository orderRepository;
```

Comment:
why do we have all this bean abundance, when we are only using one?

Reply:
Removed